templates/collections/topic_explorer_page.html

```diff
@@ -55,8 +55,8 @@ <h2 class="featured-article__heading">
         </div>
     </div>
     {% if FEATURE_RELATED_ARTICLE_ON_EXPLORE_PAGES %}
-        {% if page.related_articles %}
-            {% include "includes/related-articles-highlight-cards.html" with articles=page.related_articles title="Records revealed" %}
+        {% if page.related_record_articles %}
+            {% include "includes/related-articles-highlight-cards.html" with articles=page.related_record_articles title="Records revealed" %}
```

Comment:
Same comments as previous :)

Suggestion:
```suggestion
{% if page.related_articles %}
    <div class="row">
        <div class="col-md-12">
            {% for article_page in page.related_article %}
                <li>
                    <a href="{% pageurl article_page %}">{{ article_page.title }}</a>
                </li>
            {% endfor %}
        </div>
    </div>
```